flow/packet-headers/icmp.yaml

```diff
@@ -7,70 +7,73 @@ components:
       properties:
         choice:
           type: string
-          enum: [icmp_echo]
-        icmp_with_next_fields:
-          $ref: '#/components/schemas/Flow.ICMPEcho'
+          enum: [echo]
+        echo:
+          $ref: '#/components/schemas/Flow.Icmp.Echo'
 
-    Flow.ICMPEcho:
+    Flow.Icmp.Echo:
       x-include: 
-      - '.#/components/schemas/Flow.ICMP.NextFields'
-      description: Packet Header for ICMP Echo
+      - '.#/components/schemas/Flow.Icmp.Common'
+      - '.#/components/schemas/Flow.Icmp.NextFields'
```

Comment:
beginning . is not needed and should be removed.

Reply:
Removed the dots